src/components/VgtFilterRow.vue

```diff
@@ -162,17 +145,25 @@ export default {
       this.updateFilters(column, value);
     },
 
+    updateSlotFilter(column, value) {
+      let fieldToFilter = column.filterOptions.slotFilterField || column.field;
+      if (typeof column.filterOptions.formatValue === 'function') {
```

Comment:
This was an addition today, FYI. In working with this branch in our team project, I was formatting the filter data right before passing it in, but passing the function into the table made it much cleaner and enhanced reusability in our own codebase.